index.html

```diff
@@ -121,13 +121,13 @@ <h4>Tonic</h4>
                 </div>
               </div>
             </li>
-            <li class="work">
+            <li class="work work-reverse">
              <div class="multi-post-stories">
              <div class="tonic">
               <img src="assets/SnapshootPortfolio-4.jpg" class="img" alt="stories">
              </div>
              </div >
```

Comment:
- [ ] Kindly, let us arrange our work project to look the same as we have on the same [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

  --------------------------------------Original-----------------------------------
![project1](https://user-images.githubusercontent.com/55635977/126772423-347795fe-3576-456c-8c27-101df0473e07.png)

  
  -----------------------------------------Yours-----------------------------------

![project2](https://user-images.githubusercontent.com/55635977/126771911-6db017c7-595f-42c3-a30f-f5ae24f02f32.png)